docs/sources/configure/_index.md

```diff
@@ -3143,14 +3143,22 @@ shard_streams:
 
 # OTLP log ingestion configurations
 otlp_config:
+  # Configuration for Resource Attributes to store them as index labels or
+  # Structured Metadata or drop them altogether
   resource_attributes:
+    # Configure whether to ignore the default list of Resource Attributes to be
+    # stored as Index Labels and only use the given Resource Attributes config
```

Comment:
What is the default?

Reply:
LOL, no I didn't need the list of attributes, but what's the default for the ignore setting (Ignore or do not ignore).

Reply:
Ahh, sorry, my bad. Default is false. I will see if I can add the default value.